presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcFileRewriter.java

```diff
@@ -227,7 +227,7 @@ private static OrcFileInfo rewrite(
         return new OrcFileInfo(rowCount, uncompressedSize);
     }
 
-    private static Page maskedPage(Block[] blocks, BitSet rowsToDelete, int start, int count)
+    public static Page maskedPage(Block[] blocks, BitSet rowsToDelete, int start, int count)
```

Comment:
`rowToDelete` to `rowMask`.  This method might be better moved to a separate class `OrcUtils` since now it is shared among classes. 
Change commit message "enable delta read" to "Add rowMask to raptor orc reader". Ideally, `OrcPageSource` change can be a separate commit.